plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitManager.java

```diff
@@ -290,8 +289,8 @@ private List<DeltaLakeSplit> splitsForFile(
 
     private static String buildSplitPath(String tableLocation, AddFileEntry addAction)
     {
-        // paths are relative to the table location and URL encoded
-        String path = URLDecoder.decode(addAction.getPath(), UTF_8);
+        // paths are relative to the table location and URI
```

Comment:
Could you fix the below comment and add a link to https://github.com/delta-io/delta/blob/master/PROTOCOL.md#add-file-and-remove-file? 

https://github.com/trinodb/trino/blob/6283fef/plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java#L1258